setup.py

```diff
@@ -28,35 +26,35 @@
         "tensorboardX>=1.8",  # For pytorch<1.1.0
         # Signal processing related
         "librosa>=0.8.0",
-        "resampy",
-        "pysptk>=0.1.17",
         # Natural language processing related
         # FIXME(kamo): Sentencepiece 0.1.90 breaks backwardcompatibility?
         "sentencepiece<0.1.90,>=0.1.82",
         "nltk>=3.4.5",
-        "morfessor",
         # File IO related
         "PyYAML>=5.1.2",
         "soundfile>=0.10.2",
         "h5py>=2.10.0",
         "kaldiio>=2.17.0",
-        "youtube_dl",  # for laborotv
         # TTS related
-        "inflect>=1.0.0",
-        "unidecode>=1.0.22",
         "pyworld>=0.2.10",
-        "nnmnkwii",
         "espnet_tts_frontend",
         # ASR frontend related
-        "museval>=0.2.1",
-        "pystoi>=0.2.2",
         "nara_wpe>=0.0.5",
         "torch_complex",
         "pytorch_wpe",
+    ],
+    "recipe": [
+        "espnet_model_zoo",
```

Comment:
It is better to install model_zoo as a default since it can be used for the demo without the recipe.

Reply:
The default modules are directly imported in `espnet` or `espnet2` module and `espnet_model_zoo` is not used.
I think we should decide it according to this definite rule to avoid confusion.

- Minimal installation: `pip install espnet` 
- Train using our recipe: `pip install "espnet[recipe]"`
- Inference only: `pip install espnet_model_zoo`
  - Note that `espnet_model_zoo` depends on `espnet`.

Reply:
I see. Now I understand the point.